modules/custom/openy_daxko2/config/install/migrate_plus.migration_group.daxko.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 8f09303f-c298-4b08-8bb2-512e9c4e6a06
+langcode: en
+status: true
+dependencies: {  }
+id: daxko
+label: Daxko
+description: 'Daxko CRM system'
+source_type: 'Use APIv2 to save categories and offerings into CSV files and then migrate from them'
+module: null
```

Comment:
I see an error here. Module can't be null. There is some broken dependency needs to be fixed